server.js

```diff
@@ -25,8 +25,12 @@ app.get('/api', (req, res) => {
 });
 
 app.post('/', (req, res) => {
-  trendQuery(req.body.keyword, (results) => {
-    res.send(results);
+  trendQuery(req.body.keyword, (err, data) => {
+    if (err) {
+      console.error('Error:', err);
```

Comment:
We probably want to send the error to the client with an appropriate status code.  Likely something in the 500s.